addons/survey/tests/test_certification_flow.py

```diff
@@ -107,8 +112,12 @@ def test_flow_certification(self):
             self._answer_question(q03, "I think they're great!", answer_token, csrf_token)
             self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token, button_submit='previous')
             self._answer_question(q03, "Just kidding, I don't like it...", answer_token, csrf_token)
-            self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token)
-            self._answer_question(q05, [q05.suggested_answer_ids.ids[0], q05.suggested_answer_ids.ids[1], q05.suggested_answer_ids.ids[3]], answer_token, csrf_token)
+            self._answer_question(q04, q04.suggested_answer_ids.ids[0], answer_token, csrf_token,
+                                  submit_query_count=47, access_page_query_count=30)
+            # Not the last to not add completion flow queries to answer validation
```

Comment:
Could you clarify this comment above question 5? Does it mean that you shouldn't test query counts on the last question because of the additional completion flow? If so, why not (adapting expected number of queries)? 
Also, with a view to improving the experience of answering surveys, is it possible to optimize this additional completion flow?